website/content/en/preview/AWS/provisioning.md

```diff
@@ -243,75 +243,10 @@ spec:
 
 ### UserData
 
-In order to specify custom user data, you must include it within the AWSNodeTemplate resource. You can then reference the AWSNodeTemplate resource through `spec.providerRef` in your provisioner.
+You can control the UserData that needs to be applied to your worker nodes via the `spec.providerRef` field in your provisioner.
+Review the [Custom UserData documentation](../user-data/) to learn the necessary steps.

```

Comment:
Can we frame this page as a single "launch templates" story that talks about everything? To start, we can just include the content you have and elaborate from there.
```suggestion
Review [Custom Launch Templates](../launch-templates/) to learn the necessary steps.
```

Reply:
I called it `Custom UserData` to distinguish it from this doc https://karpenter.sh/v0.10.1/aws/launch-templates/ that's titled `Launch Templates and Custom Images`. 

Not sure what terminology would fit better.